.claude/skills/update-from-design/SKILL.md

```
@@ -0,0 +1,109 @@
+
+If a new element only appears when `app.js` runs, add it to `snapshot.py`'s
+FIXTURE. `app.js` does not run in a snapshot, so anything it draws is invisible
+to every future render until it is faked there.
+
+## 4. Deploy to the VPS
+
+The box runs an editable install from a git checkout at `/home/reader/somnia`,
+currently on `feat/apply-redesign`. Every git command there needs
+`HOME=/home/reader` or the checkout is refused as dubiously owned.
+
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)